### DISCUSSION TITLE
How to use authentik hosted behind path prefix (subdirectory)

### DISCUSSION THREAD

**Discussion by skalahonza** [goauthentik/authentik#3478] [Q&A] [answered]
I am hosting Authentik behind path prefix: /authentik
I am using [Traefik StripPrefix](https://doc.traefik.io/traefik/middlewares/http/stripprefix/)
when I curl the /authentik path I get following:
```
< HTTP/2 302
< content-type: text/html; charset=utf-8
< date: Thu, 25 Aug 2022 08:32:23 GMT
< location: /flows/-/default/authentication/?next=/
```

however the **location** should be :
`< location: /authentik/flows/-/default/authentication/?next=/`
becuase the X-Forwarded-Prefix is sent:

`X-Forwarded-Prefix: /authentik`

i could not find any mention about **X-Forwarded-Prefix** in docs or how to host authentik behind prefix

basically, is there any way to tell authentik that it's base path is not `/` but `/authentik`? or any other prefix?

**Comment by skalahonza** [comment, marked as answer]
thanks to @undergrid (https://github.com/undergrid) for pointing out the obvious in Discord.
It cannot be done.
For example this part of code https://discord.com/channels/809154715984199690/809154716507963434/1015247700763086940 relies on being hosted in the root path (/)
